spec/requests/tasks_spec.rb

```diff
@@ -0,0 +1,127 @@
+require 'rails_helper'
+
+RSpec.describe 'Tasks API' do
+
+  let!(:story) { create(:story) }
+  let!(:tasks) { create_list(:task, 20, story_id: story.id) }
+  let(:story_id) { story.id }
+  let(:id) { tasks.first.id }
+
+  # Test suite for GET /stories/:story_id/tasks
+  describe 'GET /stories/:story_id/tasks' do
```

Comment:
GET /tasks should also be tested

Reply:
should also offer a ?story_id=x filter for tasks imo, which gives the same result as /stories/:story_id/tasks

Reply:
We need filters for search only. We do not have the search implemented yet - so I would skip that for now.

Reply:
"We need filters for search only." why would that be? if I want to pull tasks related to a story, that's a filter, no matter if the user is "searching" or not, so it should imo be implemented as a filter, especially if it aids development like this would.

Reply:
If you want to pull tasks to a story, you are using GET /stories/:story_id/tasks
This is the restful way, which I prefer.

Reply:
REST has absolutely nothing do to with URLs, but with states, mutations, etc... 
For REST, /stories/:story_id/tasks is literally the same as /tasks?story_id=:story_id, as long as they both return a list of tasks filtered by story_id. From the end of consuming the API, /tasks?... is easier to implement, cause code will be able to be reused. But I don't think it matters much if we add the one the FE needs now or later since we don't need either of them rn.

Reply:
I agree, if we need parameters, we can add them later.
I really like the following resource which explains Rest:

Resource: http://www.restapitutorial.com/lessons/restfulresourcenaming.html

**Resource URI Examples**
To read a customer with Customer ID# 33245:
GET http://www.example.com/customers/33245 
The same URI would be used for PUT and DELETE, to update and delete, respectively.
[...]
To go one layer deeper in the hierarchy:
GET http://www.example.com/customers/33245/orders/8769/lineitems/1